pkgs/tools/misc/octofetch/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "octofetch";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "azur1s";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-ciBFTVdHotjifNAoXJjI9CumyK98OkMmfWMbrEldlNI=";
+  };
+
+  cargoSha256 = "sha256-Gzemm5HY6YwlxesQlil6R+34OtAeU2k7f/9+Lll3i8k=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl ];
+
+  meta = with lib; {
+    description = "Github user information on terminal";
```

Comment:
This kind of feels like a typo, maybe GitHub user information in the terminal? Or something similar?

Reply:
I've just copied that directly from the [repository](https://github.com/azur1s/octofetch)'s description, should I fix it?

Reply:
Since it seems to be upstream's intent. I think this is fine. We can always change it if upstream changes it.